lib/pages/launch_page.dart

```diff
@@ -1 +1,22 @@
-//
\ No newline at end of file
+import 'package:firebase_core/firebase_core.dart';
+import 'package:flutter/material.dart';
+import 'package:recommender_app/firebase_options.dart';
+
```

Comment:
```suggestion
import 'package:flutter/material.dart';
```

細かくて申し訳ないですが、こちら↓の2行も削除お願いします！
- L1 `import 'package:firebase_core/firebase_core.dart';`
- L3 `import 'package:recommender_app/firebase_options.dart';`